info.md

```diff
@@ -1,11 +1,17 @@
-### Project Classification
+### Project Information
 
-* <i class="fas fa-flag" style="color:#2ADA08;"></i> Flagship Project
-* <i class="fas fa-hammer" style="color:#233e81;"></i> Breaker
-* <i class="fas fa-toolbox" style="color:#233e81;"></i> Builder
+* <i class="fas fa-flag" style="font-size: 1.3em; color:#38a047;"></i>
+  <span style="font-size: 1.3em;">Flagship Project</span>
+
+### Classification
```

Comment:
`####`? Seems to look better (e.g. https://owasp.org/www-project-juice-shop/)

Reply:
Sure, I’ll tweak that.